### ISSUE TITLE
Add region landmarks to Homepage Carousels

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13516]
**Describe the bug**

- Screen Reader users must be able to navigate by landmarks, currently, there aren't many nor labelled, therefore relevant sections on the page should be identified with meaningful landmarks names.

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS).
2. Bring up the Rotor or Element List Menu.
3. Choose Landmarks view.
4. See absence of relevant landmarks.

**Expected behavior**

- Use only predefined landmarks https://wiki.wdf.sap.corp/wiki/display/spar/Landmarks
- `<cx-carousel role="region" aria-label="Carousel, Our Best selling Products">`
- `<cx-carousel role="region" aria-label="Carousel, What's New">`

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.
![carousel](https://user-images.githubusercontent.com/45409515/145847767-2f75044d-c5bd-4737-8afa-1d44bb5f26cc.png)
<img width="1207" alt="Screen Shot 2021-12-13 at 10 54 12 AM" src="https://user-images.githubusercontent.com/45409515/145847781-0802a40a-6806-4ab7-93e7-9347c1ca8bc5.png">
